sanic/cli/app.py

```diff
@@ -76,7 +76,12 @@ def run(self):
             kwargs = self._build_run_kwargs()
             app.run(**kwargs)
         except ValueError:
-            error_logger.exception("Failed to run app")
+            if os.path.isdir(self.args.module):
+                error_logger.exception(
+                    "Please attach --simple to run your app from a directory."
+                )
+            else:
```

Comment:
Why do this check here and not in the `_get_app` exception?

Reply:
didn't know we could, let me do that :)

Reply:
I would just do this check right next to the `--factory` check.

Reply:
on it :)

Reply:
i changed where it raises slightly, however I think it should be fine, let me know :)